drivers/SmartThings/zigbee-smoke-detector/fingerprints.yml

```diff
@@ -1,4 +1,9 @@
 zigbeeManufacturer:
+  - id: "LUMI/lumi.sensor_gas.acn02"
+    deviceLabel: Aqara Smart Gas Detector
```

Comment:
Hi, @Hongming6 

The `deviceLabel` is different with  what you defined in tools/localizations.